en/04_Changelogs/5.0.0.md

```diff
@@ -212,6 +217,218 @@ There were a lot more changes than just those, so you may want to also check out
 - [List of toolbar buttons available for TinyMCE 6](https://www.tiny.cloud/docs/tinymce/6/available-toolbar-buttons/)
 - [Create a plugin](https://www.tiny.cloud/docs/tinymce/6/creating-a-plugin/)
 
+## Front-end build stack upgrades {#front-end}
+
+[info]
+"Front-end" in this section refers to the javascript and css in the CMS. It doesn't have any impact on your website's public-facing front-end.
+[/info]
+
+We've upgraded the front-end build stack for the CMS, along with most of the javascript dependencies.
+
+### Webpack config
+
+The [@silverstripe/webpack-config](https://www.npmjs.com/package/@silverstripe/webpack-config) and [@silverstripe/eslint-config](https://www.npmjs.com/package/@silverstripe/eslint-config) NPM libraries, along with the build stack for all supported modules, has been updated to be compatible with node 18 and webpack 5. This will only impact you if your module or project uses one or both of those NPM packages - you will need to make sure you upfate your own dependencies to be compatible with them, along with the dependencies listed below if you use them.
```

Comment:
```suggestion
The [@silverstripe/webpack-config](https://www.npmjs.com/package/@silverstripe/webpack-config) and [@silverstripe/eslint-config](https://www.npmjs.com/package/@silverstripe/eslint-config) NPM libraries, along with the build stack for all supported modules, has been updated to be compatible with node 18 and webpack 5. This will only impact you if your module or project uses one or both of those NPM packages - you will need to make sure you update your own dependencies to be compatible with them, along with the dependencies listed below if you use them.
```

Reply:
Done